inference-engine/tests/functional/inference_engine/transformations/dropout_with_random_uniform_replacer_test.cpp

```diff
@@ -0,0 +1,149 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/pass/manager.hpp>
+#include <string>
+#include <transformations/common_optimizations/dropout_with_random_uniform_replacer.hpp>
+#include <transformations/init_node_info.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+using namespace testing;
+
+TEST(TransformationTests, DropoutWithRandomUniformReplacerCase1) {
+    std::shared_ptr<ngraph::Function> f(nullptr), f_ref(nullptr);
+    {
+        auto input = std::make_shared<ngraph::opset8::Parameter>(ngraph::element::i32, ngraph::Shape{3});
+        auto shape = std::make_shared<ngraph::opset8::ShapeOf>(input);
+        auto min_const = ngraph::opset8::Constant::create(ngraph::element::f32, ngraph::Shape{}, {0.0});
+        auto max_const = ngraph::opset8::Constant::create(ngraph::element::f32, ngraph::Shape{}, {1.0});
+        auto ru = std::make_shared<ngraph::opset8::RandomUniform>(shape,
+                                                                  min_const,
+                                                                  max_const,
+                                                                  ngraph::element::f32,
+                                                                  100,
+                                                                  200);
+        auto add_const = ngraph::opset8::Constant::create(ngraph::element::f32, ngraph::Shape{}, {30.0});
+        auto add = std::make_shared<ngraph::opset8::Add>(ru, add_const);
+        auto floor = std::make_shared<ngraph::opset8::Floor>(add);
+        auto mul_const = ngraph::opset8::Constant::create(ngraph::element::f32, ngraph::Shape{}, {25.0});
```

Comment:
why you need mul in this tests?

Reply:
It was part of initial pattern. Now it is not needed. Removed it.